thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js

```diff
@@ -1449,32 +1449,13 @@ export default Component.extend({
      * @method onRangeSelection
      * @param {Object} rangeOption - the user-selected time range to load
      */
-    onRangeSelection(timeRangeOptions) {
-      const {
-        start,
-        end,
-        value: duration
-      } = timeRangeOptions;
-
+    onRangeSelection(start, end) {
       const startDate = moment(start).valueOf();
       const endDate = moment(end).valueOf();
+
       //Update the time range option selected
       set(this, 'analysisRange', [startDate, endDate]);
       set(this, 'displayRange', [startDate, endDate]);
-      set(this, 'duration', duration);
-      // This makes sure we don't fetch if the preview is collapsed
-      if(get(this, 'showDetails') && get(this, 'dataIsCurrent')){
-        // With a new date range, we should reset the state of time series and anomalies for comparison
-        if (get(this, 'isPreviewMode')) {
-          this.setProperties({
-            anomaliesOld: [],
-            anomaliesOldSet: false,
-            anomaliesCurrent: [],
-            anomaliesCurrentSet: false
-          });
-        }
-        this._fetchAnomalies();
-      }
```

Comment:
this is no longer required?

Reply:
Here, I removed duration since it's no longer needed.  Old lines 1468-1475 were duplicate logic, so I condensed it to one check (new lines 1460-1467).  The fetchAnomalies check is now on lines 1468-1471  (new lines).

Reply:
Thanks